src/Datadog.Trace.ClrProfiler.Native/Datadog.Trace.ClrProfiler.Native.DLL.vcxproj

```diff
@@ -170,6 +181,10 @@
       <OptimizeReferences>true</OptimizeReferences>
       <ModuleDefinitionFile>.\Datadog.Trace.ClrProfiler.Native.def</ModuleDefinitionFile>
     </Link>
+    <PreBuildEvent>
+      <Command>IF EXIST $(SolutionDir)\pre-build-events-cpp.bat CALL $(SolutionDir)\pre-build-events-cpp.bat  "$(TargetPath)"  "$(TargetFileName)"  "$(TargetDir)"  "$(TargetName)"</Command>
```

Comment:
Is this called twice?

Reply:
No, it looks like it's getting called once per Debug/Release | Win32/x64 build.

Reply:
Right. It was set for some configurations but not others.